### DISCUSSION TITLE
Swagger and other URLs

### DISCUSSION THREAD

**Discussion by KaranTrivedi** [litestar-org/litestar#1561] [Q&A] [answered]
Hello, what is the URL for litestar swagger or similar docs? api:port/docs seems to not be it.

Thanks,
Karan.

**Comment by KaranTrivedi** [comment]
I answer my own question:

Paths are
URL:PORT/schema/redoc
URL:PORT/schema/docs
URL:PORT/schema/elements

I wish this was clearer in the docs...

**Comment by Goldziher** [comment]
@provinzkraut

**Comment by KaranTrivedi** [comment, marked as answer]
I think I was looking in the wrong place, my fault entirely.
